tests/asgi/test_asgi_servers.py

```diff
@@ -109,17 +109,20 @@ def _run_server_isolated(process_factory, host, port):
         server.send_signal(signal.CTRL_C_EVENT)
         try:
             server.wait(timeout=10)
-        except (KeyboardInterrupt, subprocess.TimeoutExpired):
+        except KeyboardInterrupt
             pass
+        except subprocess.TimeoutExpired:
+            server.kill()
```

Comment:
It seems like it makes sense to just go straight for the kill in this case. According to the docs, `kill()` is just an alias for `terminate()` on Windows anyway.

Reply:
From my tests, kill is not an alias for terminate, since terminate seems to be completely ignored on windows, while kill does kill the process